Content.Shared/Movement/Components/SpeedModifierContactsComponent.cs

```diff
@@ -13,10 +13,14 @@ public sealed partial class SpeedModifierContactsComponent : Component
     [AutoNetworkedField]
     public float WalkSpeedModifier = 1.0f;
 
-    [AutoNetworkedField]
     [DataField("sprintSpeedModifier"), ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
     public float SprintSpeedModifier = 1.0f;
 
+    [DataField("affectAirborne"), ViewVariables(VVAccess.ReadWrite)]
+    [AutoNetworkedField]
```

Comment:
Do you understand why this AutoNetworkedField is needed, or are you just copying from the rest of the variables?

Reply:
It's my understanding that [AutoNetworkedField] is needed when a field should be  handled by the server while being automatically replicated to clients. If it wasn't already obvious, I'm new to contributing, so please do correct me if I'm wrong.